### ISSUE TITLE
SVG icon asset

### ISSUE THREAD

**Issue by jonathanKingston** [mozilla/testpilot-containers#710]
Hey @fangshih or @ChenMorpheus in #8 we got the containers icon.

However with the latest changes in platform we would like to use the "context-fill" option that screenshots and other icons now use.

The assets provided were in PNG and we would need them in SVG to make this work. Are you able to export this?

(Sorry for the last minute bug, I completely forgot to raise this)

Thanks

**Comment by jonathanKingston** [comment]
Unless @johngruen or @groovecoder has this?

/cc luke I'm doing this to land with the SDK removal. I pulled out the theme detection code from SDK.

**Comment by groovecoder** [comment]
There are actually svg file assets in the `.zip` file at https://github.com/mozilla/testpilot-containers/issues/8#issuecomment-267934801

**Comment by groovecoder** [comment]
@jonathanKingston oh sorry, you meant the add-on & browserAction icon. Don't mind me.

**Comment by fangshih** [comment]
Hey @jonathanKingston Here are the SVG container icon
https://github.com/mozilla/testpilot-assets/blob/master/Containers/00_Assets/Container-16.svg

Also, I was wondering if we have anything chance to have this panel to match up the visual spec sometime in the future? 😃   Thanks!

![2017-08-04 12 02 52](https://user-images.githubusercontent.com/4862573/28953835-84d3bff0-790d-11e7-87eb-20858e7fa60f.png)

**Comment by jonathanKingston** [comment]
Dang I missed this, I hand crafted it myself last night hah.

@fangshih will look today at the differences.

**Comment by jonathanKingston** [comment]
TL;DR I made some changes I can't see any differences.

@fangshih I cleaned up the SVG you provided here: https://github.com/jonathanKingston/testpilot-containers/commit/78b5de3b44c53870b387bd978673fd0074de6907

On the left you can see my attempt at drawing one (which is essentially without the border and a lightly less border radius on the plus).

The differences to yours is:
- I cut out the sketch code
- I removed nodes that I think do nothing
- I changed the fill to be "context-fill #4c4c4c" which makes it use context fill from Firefox and for non supporting versions it should use the icon colour used by 54/55.
- I also added fill-opacity="context-fill-opacity" as our native icons have that but I don't think we ever use it.

For reference here is your code:
https://raw.githubusercontent.com/mozilla/testpilot-assets/master/Containers/00_Assets/Container-16.svg

**Comment by fangshih** [comment]
Got it! Thanks @jonathanKingston
